docs/components/alert.md

```diff
@@ -1,637 +1,248 @@
-
   <KAlert
-    :is-showing="warningIsOpen"
-    alert-message="Alert message that is dismissible"
-    dismiss-type="icon"
-    appearance="warning"
-    @closed="warningIsOpen = false"
+    appearance="success"
+    message="This is success alert."
   />
-
   <KAlert
-    :is-showing="warningIsOpen2"
-    alert-message="I'm an alert"
     appearance="warning"
-    @closed="warningIsOpen2 = false"
+    message="This is warning alert."
   />
-
   <KAlert
-    alert-message="I'm a banner type alert"
-    type="banner"
-    appearance="warning"
+    appearance="danger"
+    message="This is danger alert."
   />
 </div>
 
 ```html
 <KAlert
-  alert-message="Warning alert message"
-  appearance="warning"
+  appearance="info"
+  message="This is info alert."
 />
-
 <KAlert
-  :is-showing="isShowing"
-  alert-message="Alert message that is dismissible"
-  dismiss-type="icon"
-  appearance="warning"
-  @closed="isShowing = false"
+  appearance="success"
+  message="This is success alert."
 />
-
 <KAlert
-  :is-showing="isShowing"
-  alert-message="I'm an alert"
   appearance="warning"
-  @closed="isShowing = false"
+  message="This is warning alert."
 />
-
 <KAlert
-  alert-message="I'm a banner type alert"
-  type="banner"
-  appearance="warning"
+  appearance="danger"
+  message="This is danger alert."
 />
 ```
 
-- `success`
-
-<div>
-  <KAlert alert-message="Success alert message" appearance="success" />
-
-  <KAlert
-    :is-showing="successIsOpen"
-    alert-message="Alert message that is dismissible"
-    dismiss-type="icon"
-    appearance="success"
-    @closed="successIsOpen = false"
-  />
-
-  <KAlert
-    :is-showing="successIsOpen2"
-    alert-message="I'm an alert"
-    appearance="success"
-    @closed="successIsOpen2 = false"
-  />
-
-  <KAlert
-    alert-message="I'm a banner type alert"
-    type="banner"
-    appearance="success"
-  />
-</div>
-
-```html
-<KAlert
-  alert-message="Success alert message"
-  appearance="success"
-/>
+### title
 
-<KAlert
-  :is-showing="isShowing"
-  alert-message="Alert message that is dismissible"
-  dismiss-type="icon"
-  appearance="success"
-  @closed="isShowing = false"
-/>
+A prop to provide a title string. Good for longer messages.
 
 <KAlert
-  :is-showing="isShowing"
-  alert-message="I'm an alert"
-  appearance="success"
-  @closed="isShowing = false"
+  title="Important Update On How We Handle Third-Party Cookies"
+  appearance="warning"
+  message="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
 />
 
+```html
 <KAlert
-  alert-message="I'm a banner type alert"
-  type="banner"
-  appearance="success"
+  title="Important Update On How We Handle Third-Party Cookies"
+  appearance="warning"
+  message="Lorem ipsum dolor sit amet..."
 />
 ```
 
-- `danger`
+### hideIcon
```

Comment:
See my comment on this prop in the KAlert.vue file (the description below would need to be updated as well)